aws-ebs-csi-driver.yaml

```
@@ -32,15 +32,15 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: e6dae6132fc8383340024440fc639a135bebd4ba
 
-  - runs: |
-      # Our global LDFLAGS conflict with a Makefile parameter
-      unset LDFLAGS
-
-      make ARCH=$(go env GOARCH)
-      mkdir -p ${{targets.destdir}}/usr/bin
-      cp bin/aws-ebs-csi-driver ${{targets.destdir}}/usr/bin/
-
-  - uses: strip
+  - uses: go/build
+    with:
+      packages: ./cmd
+      ldflags: |
+        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.driverVersion=${{package.version}}
+        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud.driverVersion=${{package.version}}
+        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.gitCommit=$(git rev-parse HEAD)
+        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.buildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
```

```suggestion
        -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.buildDate=$(date -u "+%Y-%m-%dT%H:%M:%SZ")
```

I think we should change this as well. the build time should be build time when the package is built. 
```bash
 aws-ebs-csi-driver --version
{
  "driverVersion": "1.36.0",
  "gitCommit": "e6dae6132fc8383340024440fc639a135bebd4ba",
  "buildDate": "1970-01-01T00:00:00Z",
  "goVersion": "go1.23.2",
  "compiler": "gc",
  "platform": "linux/amd64"
}
```

upstream image: 
```bash
$ docker run --rm registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.36.0 --version
{
  "driverVersion": "v1.36.0",
  "gitCommit": "",
  "buildDate": "2024-10-14T19:01:44+00:00",
  "goVersion": "go1.23.2",
  "compiler": "gc",
  "platform": "linux/amd64"
}
```

---

I think this is something we do it on purpose for providing reproducibility, so I don't think that we should change it brother